odl/phantom/misc_phantoms.py

```diff
@@ -146,6 +146,82 @@ def rect(x):
     return out.ufuncs.minimum(1, out=out)
 
 
+def text(space, text, font='arial', scale=0.8, inverted=True):
+    """Create phantom of text.
+
+    The text is represented by an scalar image taking values in [0, 1].
+    Depending on the choice of font, the text may or may not be anti-aliased.
+    anti-aliased text can take any value between 0 and 1, while
+    non-anti-aliased text takes values exclusively in {0, 1}.
+
+    This method requires the PIL package.
+
+    Parameters
+    ----------
+    space : `DiscreteLp`
+        Discretized space in which the phantom is supposed to be created.
+        Must be two-dimensional.
+    text : str
+        The text that should be written in the phantom.
+    font : str, optional
+        The font that should be used to write the text. Available options are
+        platform dependent.
+    scale : float, optional
+        Scaling of the text. 1.0 indicates that the phantom should occupy all
+        of the space along its largest dimension.
+    inverted : bool, optional
+        If the phantoms should be given in inverted style, i.e. white on black.
+
+    Returns
+    -------
+    phantom : ``space`` element
+        The text phantom in ``space``.
+    """
+    from PIL import Image, ImageDraw, ImageFont
+
+    if space.ndim != 2:
+        raise ValueError('`space` must be two-dimensional')
+
+    text = str(text)
+
+    # Figure out what font size we should use by creating a very fine font
+    # and calculating its size
+    init_size = 1000
```

Comment:
It is re-used a few lines below, trying to not violate DRY and also avoid magic numbers, so this will stay.